feat: move reporter from https://github.com/re-gor/porchmark-pretty-reporter

alekzonder opened the pull request:
Перетащил и причесал репортер в packages/porchmark-pretty-reporter

Если @re-gor будет против, можем выпилить и изменить у него

re-gor committed:
Initial commit

re-gor committed:
Create essential-check.yml

Commit:
Let's get it started

Commit:
pre publish

Commit:
Expose types. Fix `saveToFs`

dependabot[bot] committed:
Bump lodash from 4.17.15 to 4.17.19

Bumps [lodash](https://github.com/lodash/lodash) from 4.17.15 to 4.17.19.
- [Release notes](https://github.com/lodash/lodash/releases)
- [Commits](https://github.com/lodash/lodash/compare/4.17.15...4.17.19)

Signed-off-by: dependabot[bot] <support@github.com>

re-gor committed:
Merge pull request #1 from re-gor/dependabot/npm_and_yarn/lodash-4.17.19

Bump lodash from 4.17.15 to 4.17.19

mutantcornholio reviewed (changes requested):
1. Давай сначала узнаем, будет ли @re-gor против, а потом будем забирать код
2. Как насчёт скопировать код через `git subtree`, чтобы сохранить все коммиты и авторов, а потом сверху сделать работу по интеграции?

alekzonder commented:
Он в отпуске, узнаем недели через 3

alekzonder committed:
Add 'src/packages/porchmark-pretty-reporter/' from commit '795f2338d5d7b4b5c5aff8dae8cd7c1a9f2e623c'

git-subtree-dir: src/packages/porchmark-pretty-reporter
git-subtree-mainline: 993848a548ecad384d972e7df1da6d69142eac01
git-subtree-split: 795f2338d5d7b4b5c5aff8dae8cd7c1a9f2e623c

alekzonder committed:
chg: packages/porchmark-pretty-reporter move src

alekzonder committed:
fix: eslint and ts errors

alekzonder committed:
chg: restore ChartReport, fix eslint config, add deps

alekzonder commented:
@mutantcornholio /ping

mutantcornholio reviewed (approved):
найс
